lib/utils/room_status_extension.dart

```diff
@@ -120,4 +85,55 @@ extension RoomStatusExtension on Room {
     }
     return lastReceipts.toList();
   }
+
+  String _getLocalizedStatusGroupChat(BuildContext context) {
+    final totalMembers =
+        (summary.mInvitedMemberCount ?? 0) + (summary.mJoinedMemberCount ?? 0);
+
+    return L10n.of(context)!.membersCount(totalMembers.toString());
+  }
+
+  String _getLocalizedStatusDirectChat(
+    CachedPresence? presence,
+    BuildContext context,
+  ) {
+    final directChatPresence = presence ?? this.directChatPresence;
+
+    if (directChatPresence != null) {
+      if (directChatPresence.presence == PresenceType.online) {
+        return L10n.of(context)!.onlineStatus;
+      }
+      final time = directChatPresence.lastActiveTimestamp;
+
+      if (time != null) {
+        if (DateTime.now().isBefore(time.add(const Duration(hours: 1)))) {
+          return L10n.of(context)!
+              .onlineMinAgo(DateTime.now().difference(time).inMinutes);
+        } else if (DateTime.now()
+            .isBefore(time.add(const Duration(hours: 24)))) {
+          final timeOffline = DateTime.now().difference(time);
+          return L10n.of(context)!.onlineHourAgo(
+            timeOffline.inHours,
+            timeOffline.inMinutes - (timeOffline.inHours * 60),
+          );
+        } else if (DateTime.now().isBefore(time.add(const Duration(days: 7)))) {
+          final timeOffline = DateTime.now().difference(time);
+          return L10n.of(context)!.onlineDayAgo(timeOffline.inDays);
+        } else if (DateTime.now()
+            .isBefore(time.add(const Duration(days: 30)))) {
+          final timeOffline = DateTime.now().difference(time);
+          return L10n.of(context)!
+              .onlineWeekAgo((timeOffline.inDays / 7).truncate());
+        } else if (DateTime.now()
+            .isBefore(time.add(const Duration(days: 365)))) {
+          final timeOffline = DateTime.now().difference(time);
+          return L10n.of(context)!
+              .onlineMonthAgo((timeOffline.inDays / 30).truncate());
+        } else {
+          return L10n.of(context)!.onlineLongTimeAgo;
+        }
```

Comment:
IMO, it is `offline` too

Reply:
no because here the date is more than 1 year